locales/pl_PL/LC_MESSAGES/duckduckgo.po

```diff
@@ -1228,7 +1228,7 @@ msgstr "Rozrywka"
 
 msgctxt "SERP footer content"
 msgid "Escape The Filter Bubble"
-msgstr ""
+msgstr "Wyrwij się z Bańki Filturjącej"
```

Comment:
I never heard anyone talk like that, but that's how wikipedia translates it: https://pl.wikipedia.org/wiki/Ba%C5%84ka_filtruj%C4%85ca

Reply:
Nie przetłumaczyłbym tego w ten sposób. To nie bańka filtruje, tylko bańka jest spowodowana przez filtry. Wikipedia też może się mylić.

Reply:
I'd also consider "bańka informacyjna"/"bańka informacji"

Reply:
oh I like "bańka informacyjna" and it seems to be widely used 👍